packages/playground/src/components/user-management/form-user-delete-dialog.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+
+import {
+  Spacer,
+  AlertDialog,
+  AlertDialogTrigger,
+  AlertDialogContent,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogCancel,
+  Button,
+  Icon
+} from '@harnessio/canary'
+
+interface UsersProps {
+  uid: string
+  display_name?: string // Add a default value of undefined
+}
+
+interface FormResetPasswordrDialogProps {
+  user: UsersProps
+  onClose: () => void
+  onDelete: () => void
+  isDeleting: boolean
+  deleteSuccess: boolean
+}
+
+//Form Delete Member Dialog
+export const FormDeleteUserDialog: React.FC<FormResetPasswordrDialogProps> = ({
+  user,
+  onClose,
+  onDelete,
+  isDeleting,
+  deleteSuccess
+}) => {
+  // const [isDeleting, setIsDeleting] = useState(false)
+  // const [deleteSuccess, setDeleteSuccess] = useState(false)
+  // // Delete project handler
+  // const handleDelete = () => {
+  //   setIsDeleting(true)
+  //   setTimeout(() => {
+  //     setIsDeleting(false)
+  //     setDeleteSuccess(true) // Mark deletion as successful
+  //     setTimeout(() => {
+  //       onClose() // Close the dialog
+  //     }, 2000)
+  //   }, 2000)
+  // }
```

Comment:
do u need this

Reply:
This is only show an example originally for description if we use useState Hook that handle event in the children component.

Reply:
Will clean up later after discussion. Thanks for reminding me